packages/gestalt/src/ListItem.js

```diff
@@ -0,0 +1,53 @@
+// @flow strict
+import { type Element, type Node } from 'react';
+import classnames from 'classnames';
+import Text from './Text.js';
+import ListText from './ListText.js';
+import styles from './List.css';
+import getChildrenToArray from './List/getChildrenToArray.js';
+import { useList } from './contexts/ListProvider.js';
+
+type Props = {|
+  /**
+   * Use List.NestedList to build nested lists. See [subcomponents](/web/list#Subcomponents).
```

Comment:
This link needs to use a full URL. Also, wouldn't linking to the "nesting" variant be more useful?